dev-php/ca-bundle/ca-bundle-1.0.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Find a path to the system CA bundle, and fallback to the Mozilla CA bundle"
+HOMEPAGE="https://github.com/composer/ca-bundle"
+
```

Comment:
Someone on IRC point me at the skel ebuild (https://gitweb.gentoo.org/repo/gentoo.git/tree/skel.ebuild) and there is quite some blank line there, nearly after each definition of variable,
I delete the blank line as you suggested but I am a bit confuse about that.